docs/source/dataset.rst

```diff
@@ -0,0 +1,42 @@
+===============
+Sample Dataset
+===============
+Public datasets
+~~~~~~~~~~~~~~~~
+Some data to work with have been obtained from `Zenodo <https://zenodo.org>`_. 
+
++-------------------------------------------------+----------+---------+----------+----------+
+|   Title                                         |Download  |  Format | Dimension| Size     |
++=================================================+==========+=========+==========+==========+
+| `Bullseye probe`_                               |`link1`_  | HDF5    | 4D       | 2.1 GB   |             
++-------------------------------------------------+----------+---------+----------+----------+
+| `Circular probe`_                               |`link2`_  |  HDF5   |  4D      | 2.1 GB   |
++-------------------------------------------------+----------+---------+----------+----------+
+| `Electron Bessel beam diffraction pattern`_     |`link3`_  | DM3     | 3D       | 2.6 GB   |
++-------------------------------------------------+----------+---------+----------+----------+
+Creating your own dataset
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+Random data can be generated in the following way. It should be kept in mind that the data generated in this way can only be used for simple testing as it has no physical significance.
+
+.. code-block:: python
+      
+      import numpy as np
+      real_data = np.random.randn(16, 16, 16, 16).astype("float32")
+      real_data.tofile("/tmp/real_raw_file.raw")
+
+Now you can load the data through the :ref:`Python API <api>` in the following way
```

Comment:
Sorry I didn't saw this earlier. If you are trying to reference [Python API](https://libertem.github.io/LiberTEM/api.html) , you need to mention label placed above the section title.

For example : 
 `.. _my-reference-label:`

`Section to cross-reference`
`==========================`
This can be referenced by  :ref:\`my-reference-label\`

Reply:
My bad, will fix them asap.